spec/presenters/world_index_graphql_presenter_spec.rb

```diff
@@ -0,0 +1,93 @@
+RSpec.describe WorldIndexGraphqlPresenter do
+  let(:fixture) { fetch_graphql_fixture("world_index").dig("data", "edition") }
+  let(:content_item) { WorldIndexGraphql.new(fixture) }
+  let(:world_index_presenter) { described_class.new(content_item) }
```

Comment:
**suggestion:** test performance

Can we memoise this somehow to avoid (re)loading the fixture on every test?

Reply:
Good spot.  I'd forgotten that `let` only caches for the duration of each test.

I've updated this and also the other non-GraphQL test to behave the same way.